packages/smart-wallet/src/invitations.js

```diff
@@ -4,6 +4,9 @@ import { fit } from '@agoric/store';
 import { E } from '@endo/far';
 import { shape } from './typeGuards.js';
 
+// Ambient types. Needed only for dev but this does a runtime import.
+import '@agoric/zoe/src/types.js';
```

Comment:
this bloats the smart-wallet contract vat too, doesn't it? not that much, I suppose.

Reply:
the whole file is comments. does the vat bundler leave comments?

Reply:
yes. it tries not to change the source at all.

The Zoe security model depends, in some cases, on static analysis of contract source; in particular, tracing invitation descriptions to the code that will execute when the invitations are exercised. (IOU pointer to docs; I'm not certain they exist, though.)